_episodes/05-function-style.md

```diff
@@ -14,41 +14,58 @@ keypoints:
 ---
 
 # Adding a function to our package
-We will now add a new function to our Python package. Open your `molssi_math.py` module in a text editor. Currently, it should only have the default `canvas` function written by the CMS CookieCutter.
+We will now add a new function to our Python package. Create a new folder named `io` under `molecool` directory and add a file named `pdb.py`. Open your `molecool/io/pdb.py` module in a text editor. Currently, it should only have the default `canvas` function written by the CMS CookieCutter.
 
-Let's add another function. We will add a function to calculate the mean of a list.
+Let's add another function. We will add a function to open and read coordinates and atom symbols from a pdb file.
 
 ~~~
-def mean(num_list):
-    mean_list = sum(num_list)/len(num_list)
-    return mean_list
+def open_pdb(f_loc):
+    with open(f_loc) as f:
+        data = f.readlines()
+    c = []
+    sym = []
+    for l in data:
+        if 'ATOM' in l[0:6] or 'HETATM' in l[0:6]:
+            sym.append(l[76:79].strip())
+            c2 = [float(x) for x in l[30:55].split()]
+            c.append(c2)
+    coords = np.array(c)
+    return sym, coords
 ~~~
 {: .language-python}
 
 This function should now be accessible when we execute it in the interactive Python interpreter. Test this by opening the interactive Python interpreter and typing the following
 
 ~~~
->>> import molssi_devops as md
->>> md.mean([1, 2, 3, 4, 5])
+>>> import os
+>>> from molecool.io import open_pdb
+>>> pdb_file = os.path.join('molecool', 'data', 'pdb', 'water.pdb')
+>>> symbols, coords = open_pdb(pdb_file)
+>>> symbols
 ~~~
 {: .language-python}
 
 ~~~
-3.0
+array(['O', 'H', 'H'], dtype='<U1')
 ~~~
 {: .output}
 
-You should get a value of 3.0 from executing this code.
+You should get a numpy array with atomic sybols of the water molecule from executing this code.
+You can also see the atomic coordinates by executing:
+~~~
+>>> coords
+~~~
+{: .language-python}
 
 Hooray! We've successfully implemented and are able to use our own custom function! However, if we compare what we've written above to the sample function, `canvas`, we notice that ours is a little different.
 
 Keeping your previous Python interpreter open, type the following:
 
 ~~~
-help(md.canvas)
+help(open_pdb)
```

Comment:
This should still be canvas to demonstrate that the presence of a doctoring results in help being displayed.